File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    l = s.lower()
    a =[]
    for str in l.split(" "):
        if(len(str)>1):
            a.append(str[0].upper()+str[1:])
        else:
            a.append(str.upper())
    return " ".join(a)

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    const l = s.toLowerCase()
    let a = []
    for(str of l.split(" ")){
        len = str.length
        if(len >1){
            a.push(str[0].toUpperCase()+str.slice(1,len))
        }else{
            a.push(str.toUpperCase())
        }
    }
    return a.join(" ")
}